<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>594</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::PreventContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>231</width>
      <height>531</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="title">
     <string/>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QLabel" name="label_state">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>191</width>
       <height>51</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::WinPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="lineWidth">
      <number>20</number>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_start">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>启动</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_stop">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>80</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>停止</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_sync">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>发送</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_reset">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>130</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>复位</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumber">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>210</y>
       <width>191</width>
       <height>51</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::WinPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="smallDecimalPoint">
      <bool>false</bool>
     </property>
     <property name="numDigits">
      <number>4</number>
     </property>
     <property name="digitCount">
      <number>4</number>
     </property>
     <property name="mode">
      <enum>QLCDNumber::Dec</enum>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Flat</enum>
     </property>
     <property name="value" stdset="0">
      <double>12.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>190</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>运行时间</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>541</width>
      <height>261</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QGraphicsView" name="graphicsView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>521</width>
       <height>241</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>280</y>
      <width>541</width>
      <height>261</height>
     </rect>
    </property>
    <property name="title">
     <string>当前信息</string>
    </property>
    <widget class="QTableView" name="tableView_method">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>521</width>
       <height>231</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>方法</string>
    </property>
    <addaction name="create"/>
    <addaction name="open"/>
    <addaction name="save"/>
    <addaction name="saveas"/>
    <addaction name="close"/>
    <addaction name="action_6"/>
    <addaction name="action_15"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>设置</string>
    </property>
    <addaction name="action_7"/>
    <addaction name="action_8"/>
    <addaction name="action_14"/>
    <addaction name="action_16"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>采集</string>
    </property>
    <addaction name="action_9"/>
    <addaction name="actionCans"/>
    <addaction name="action_10"/>
    <addaction name="action_11"/>
    <addaction name="action_U"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>监控</string>
    </property>
    <addaction name="action_12"/>
    <addaction name="action_13"/>
   </widget>
   <widget class="QMenu" name="ParamSetting">
    <property name="title">
     <string>参数设定</string>
    </property>
    <addaction name="separator"/>
    <addaction name="action_FID"/>
    <addaction name="action_TCD"/>
    <addaction name="separator"/>
    <addaction name="qihuashi"/>
    <addaction name="fuzhuwenxiang"/>
    <addaction name="waishimingmin"/>
    <addaction name="zhuwenxiang"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="ParamSetting"/>
   <addaction name="menu_4"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="create">
   <property name="text">
    <string>新建</string>
   </property>
  </action>
  <action name="open">
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="save">
   <property name="text">
    <string>保存</string>
   </property>
  </action>
  <action name="saveas">
   <property name="text">
    <string>另存为</string>
   </property>
  </action>
  <action name="close">
   <property name="text">
    <string>关闭方法</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>最近方法</string>
   </property>
  </action>
  <action name="action_7">
   <property name="text">
    <string>串口设置</string>
   </property>
  </action>
  <action name="action_8">
   <property name="text">
    <string>网络设置</string>
   </property>
  </action>
  <action name="action_9">
   <property name="text">
    <string>日志数据</string>
   </property>
  </action>
  <action name="actionCans">
   <property name="text">
    <string>cans数据</string>
   </property>
  </action>
  <action name="action_10">
   <property name="text">
    <string>网络监控数据</string>
   </property>
  </action>
  <action name="action_11">
   <property name="text">
    <string>实时控制数据</string>
   </property>
  </action>
  <action name="action_12">
   <property name="text">
    <string>实时数据</string>
   </property>
  </action>
  <action name="action_13">
   <property name="text">
    <string>曲线绘制</string>
   </property>
  </action>
  <action name="action_U">
   <property name="text">
    <string>自动更新数据到U盘</string>
   </property>
  </action>
  <action name="action_14">
   <property name="text">
    <string>存储设置</string>
   </property>
  </action>
  <action name="action_15">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="action_16">
   <property name="text">
    <string>绘图设置</string>
   </property>
  </action>
  <action name="action_FID">
   <property name="text">
    <string>检测器FID </string>
   </property>
  </action>
  <action name="action_TCD">
   <property name="text">
    <string>检测器TCD</string>
   </property>
  </action>
  <action name="qihuashi">
   <property name="text">
    <string>汽化室</string>
   </property>
  </action>
  <action name="fuzhuwenxiang">
   <property name="text">
    <string>辅助温箱</string>
   </property>
  </action>
  <action name="waishimingmin">
   <property name="text">
    <string>外事命名</string>
   </property>
  </action>
  <action name="zhuwenxiang">
   <property name="text">
    <string>柱温箱设定</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
